h/browser/chrome/test/sidebar-injector-test.js

```diff
@@ -78,21 +82,33 @@ describe('SidebarInjector', function () {
   }
 
   describe('.injectIntoTab', function () {
-    var protocols = ['chrome:', 'chrome-devtools:', 'chrome-extension:'];
-    protocols.forEach(function (protocol) {
-      it('bails early when trying to load an unsupported ' + protocol + ' url', function () {
-        var spy = fakeChromeTabs.executeScript;
-        var url = protocol + '//foo/';
-
-        return injector.injectIntoTab({id: 1, url: url}).then(
-          assertReject, function (err) {
-            assert.instanceOf(err, errors.RestrictedProtocolError);
-            assert.notCalled(spy);
-          }
-        );
-      });
+    var urls = [
+      'chrome://version',
+      'chrome-devtools://host',
+      'chrome-extension://1234/foo.html',
+      'chrome-extension://1234/foo.pdf',
+    ];
+    urls.forEach(function (url) {
+      it('bails early when trying to load an unsupported url: ' + url,
+        function () {
+          var spy = fakeChromeTabs.executeScript;
+          return injector.injectIntoTab({id: 1, url: url}).then(
+            assertReject, function (err) {
+              assert.instanceOf(err, errors.RestrictedProtocolError);
+              assert.notCalled(spy);
+            }
+          );
+        });
     });
 
+    it('succeeds if the tab is already displaying the embedded PDF viewer',
+      function () {
```

Comment:
Small thing, but we should probably be consistent about:

```javascript
it('does a thing', function () {
  ...
});
```

vs

```javascript
it('does a thing', 
  function () {
    ...
  }
);
```

I'm not sure if you did this because of line length? If so, *seriously* don't worry about it. Especially in tests.